transformers/component_tests/queries/all_urns_query_test.go

```diff
@@ -268,7 +268,8 @@ var _ = Describe("All Urns function", func() {
 			fakeHeaderTwoID, err := headerRepo.CreateOrUpdateHeader(fakeHeaderTwo)
 			Expect(err).NotTo(HaveOccurred())
 
-			err = vatRepo.Create(diffID, fakeHeaderTwoID, metadata.UrnInk, strconv.Itoa(updatedInk))
+			diffIdTwo := test_helpers.CreateFakeDiffRecordWithHeader(db, fakeHeaderTwo)
```

Comment:
Why did you have to add these?

Reply:
I added these storage_diff records for each urn that was created since `delete_obsolete_urn_snapshot` (and other functions associated with this trigger) are now getting block numbers from the diff record, rather than the header. And so, we needed to have a new diff so that the functions could find the block number that we're expecting, so the correct snapshot would be deleted.

Reply:
👍